lib/HooksWorkerClient.js

```diff
@@ -57,7 +57,7 @@ class HooksWorkerClient {
   }
 
   terminateHandler(callback) {
-    logger.verbose('Terminating hooks handler process, PID', this.handler.pid);
+    logger.debug('Terminating hooks handler process, PID', this.handler.pid);
```

Comment:
Some places we're adding fullstop and quotes around variables in the message and other times relying on the logger behaviour of multiple arguments. Should we have some consistency here?

```
logger.debug(`Exiting Dredd process with status '${exitStatus}'.`);
```

Reply:
Good point, will do in a separate PR. It doesn't seem that anyone ever thought about consistency in that.